desktop-cosmic/cosmic-app-library/spec

```diff
@@ -0,0 +1,4 @@
+VER=1.0.0-alpha.6
+SRCS="tbl::https://github.com/pop-os/cosmic-applibrary/archive/refs/tags/epoch-${VER}.tar.gz"
+CHKSUMS="sha256::0dfc7a4f1ae4c4bb19de138a686bd34798fe4dc9767b57f459ef8f3b6974eb2d"
+CHKUPDATE="github::repo=pop-os/cosmic-applibrary"
```

Comment:
Please use Anitya where possible. GitHub API is not usable for our automated tools (such as `autopr` and BuildIt!'s `/bump` command).

Reply:
Same for all below.